app/lib/f13614c_pdf.rb

```diff
@@ -5,6 +5,14 @@ def source_pdf_name
     "f13614c-TY2021"
   end
 
+  def document_type
```

Comment:
I _think_ I'm more in favor of deleting the `DocumentTypes::Form13614C` class since this isn't a document the client uploads. Your way is OK too.